swagger.yaml

```
@@ -863,6 +863,428 @@ paths:
+    - $ref: "#/components/parameters/guestId"
+    get:
+      summary: 対象者の行動ログの取得
+      tags:
+      - general
+      security:
+      - admin: []
+      responses:
+        200:
+          description:
+            対象者の行動ログ
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/activityLog"
+        404:
+          description: ID に該当する Guest が存在しない
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /exh/status/:
+    get:
+      summary: 全展示・校内の情報をの取得
+      tags:
+      - exhibition
+      security:
+      - admin: []
+      responses:
+        200:
+          description:
+            各展示・校内の情報と滞在状況
+          content:
+            application/json:
+              schema:
+                type: object  
+                properties:
+                  exh:
+                    type: object
+                    additionalProperties:
+                      $ref: "#/components/schemas/exhStatus"
+                  all:
+                    type: object
+                    allOf:
+                    - $ref: "#/components/schemas/exhStatus"
+                    - properties:
+                        room_id:
+                          writeOnly: true
+                      
+  /exh/status/{id}/:
+    parameters:
+      - $ref: "#/components/parameters/exhibitionId"
+    get:
+      summary: 展示の情報をの取得
+      tags:
+      - exhibition
+      security:
+      - blogWriter: []
+      responses:
+        200:
+          description:
+            展示の情報と滞在状況
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/exhStatus"
+        404:
+          description: 該当する展示が存在しない
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /exh/enter/:
+    post:
+      summary: 展示入室処理
+      tags:
+      - exhibition
+      security:
+      - blogWriter: []
+      requestBody:
+        description: 入室者の情報
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                guest_id:
+                  type: string
+      responses:
+        200:
+          description: |
+            展示の滞在者数に関する情報
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/exhStatus"
+        400:
+          description: |  
+            status code とエラー内容が下記の通り対応
+            | Status Code | Explanation |
+            | :----------|  :-- |
+            | `GUEST_NOT_FOUND` | 該当する Guest が存在しない |
+            | `GUEST_ALREADY_ENTERED` | この展示にすでに入室済みでまだ退室していない |
+            | `PEOPLE_LIMIT_OVER` | 人数制限に達した |
+            | `GUEST_ALREADY_EXITED` | すでに退場済みである |
+            | `EXIT_TIME_OVER` | 退場予定時刻を過ぎている |
```

(近いからくっつけました(最上川))
OVER より exceeded のがよさそう
```suggestion
            | `PEOPLE_LIMIT_EXCEEDED` | 人数制限に達した |
            | `GUEST_ALREADY_EXITED` | すでに退場済みである |
            | `EXIT_TIME_EXCEEDED` | 退場予定時刻を過ぎている |
```